syntax = "proto3";

package Mine;


service Mine
{
    rpc FizzBuzz(FizzBuzzRequest) returns (FizzBuzzResponse);
    rpc Count(UnsignedInteger) returns (stream UnsignedInteger);
}

message FizzBuzzRequest
{
    int32 i = 1;
}

message FizzBuzzResponse
{
    string s = 1;
}

message UnsignedInteger
{
    int32 u = 1;
}
